libs/sdk-ui-ext/src/internal/components/configurationPanels/BulletChartConfigurationPanel.tsx

```diff
@@ -34,6 +34,11 @@ export default class BulletChartConfigurationPanel extends ConfigurationPanelCon
         const xAxisNameSectionDisabled = controlsDisabled || itemsOnXAxis !== 1;
         const isNameSubsectionVisible: boolean = featureFlags.enableAxisNameConfiguration as boolean;
         const isAxisLabelsFormatEnabled: boolean = featureFlags.enableAxisLabelFormat as boolean;
+        const isAxisNameViewByTwoAttributesEnabled: boolean =
+            featureFlags?.enableAxisNameViewByTwoAttributes as boolean;
```

Comment:
Nitpick: the `?.` is not necessary, if `featureFlags` is null the previous lines would already blow up

Reply:
Removed.